music21/stream/base.py

```diff
@@ -486,7 +490,7 @@ def __getitem__(self,
         3.0
 
 
-        If a class is given then an iterator of elements
+        If a class (or iterable of classes) is given, then an iterator of elements
```

Comment:
better to demo these two things separately.  `If a class is given, then a RecursiveIterator of elements matching the class is returned, similar...`

then later, "Multiple classes can be passed in separated by commas.  Any element matching any of the classes will be returned."    with the demo of that.